Makefile

```diff
@@ -1,8 +1,10 @@
 .PHONY: build generate-completion test test-cover view-cover
 
 TAG=$(shell git describe --tags 2> /dev/null || echo unknown)
+GIT_COMMIT=$(shell git rev-parse HEAD 2> /dev/null || echo unknown)
+CLC_VERSION=$(shell git tag | sort -V | tail -1 2> /dev/null || echo unknown)
 CLIENT_TYPE="CLC"
-LDFLAGS="-X 'github.com/hazelcast/hazelcast-go-client/internal.ClientType=$(CLIENT_TYPE)' -X 'github.com/hazelcast/hazelcast-go-client/internal.ClientVersion=$(TAG)'"
+LDFLAGS="-X 'github.com/hazelcast/hazelcast-go-client/internal.ClientType=$(CLIENT_TYPE)' -X 'github.com/hazelcast/hazelcast-commandline-client/internal.GitCommit=$(GIT_COMMIT)' -X 'github.com/hazelcast/hazelcast-go-client/internal.ClientVersion=$(TAG)' -X 'github.com/hazelcast/hazelcast-commandline-client/internal.ClientVersion=$(CLC_VERSION)'"
```

Comment:
Let's use the same version number for both `github.com/hazelcast/hazelcast-go-client/internal.ClientVersion` and `github.com/hazelcast/hazelcast-commandline-client/internal.ClientVersion`

Reply:
Could you also remove `CLIENT_TYPE="CLC"` line and use `CLC` directly at `github.com/hazelcast/hazelcast-go-client/internal.ClientType=`?

Reply:
For the first comment, should we remove `github.com/hazelcast/hazelcast-commandline-client/internal.ClientVersion`and just use version number of Go Client which is `1.3.0`?

Reply:
We override the Go client version with CLC version in order to be able to show the correct version on MC, so we should use the CLC version, not Go Client version.

Reply:
Fixed at https://github.com/hazelcast/hazelcast-commandline-client/pull/107/commits/4d6e7149dd84650c120e2cd2d9323b80087d1ad9.